cime_config/config_grids.xml

```diff
@@ -2771,6 +2828,13 @@
       <desc>ECwISC30to60E3r2 is a MPAS ocean grid generated with the jigsaw/compass process using the eddy closure density function that has 30 km gridcells at the equator, 60 km at mid-latitudes, and 35 km at high latitudes. Additionally, it has ocean in ice-shelf cavities:</desc>
     </domain>
 
+    <domain name="IcoswISC30E3r5">
+      <nx>465044</nx>
+      <ny>1</ny>
+      <file grid="ice|ocn">$DIN_LOC_ROOT/share/domains/domain.ocn.IcoswISC30E3r5.231121.nc</file>
+      <desc>IcoswISC30E3r5 is a MPAS ocean grid generated with the jigsaw/compass process XXX. Additionally, it has ocean in ice-shelf cavities:</desc>
```

Comment:
Should "XXX" be replaced with something?

Reply:
Thanks for the reminder @rljacob. I'll pull the description from the PR that was closed and add it here